demo/bulk_insert/bulk_insert.py

```diff
@@ -118,9 +135,18 @@ def ProcessNodes(nodes_csv_files):
 			# The first row of a label CSV contains its attributes
 			descriptor.attributes = header_row
 
+
 			# Update counts
 			header_len = len(header_row)
 			descriptor.attribute_count += header_len
+
+			# Count number of entities (line count - 1 for header row)
+			descriptor.count_entities(csvfile)
+
+			# Skip header row
+			reader.next()
+			print 'Inserting Label "%s" - %d nodes' % (label_name, descriptor.total_entities)
+
```

Comment:
Line 155, labels.descriptors.append(descriptor)
			labels.type_count += 1

Isn't labels.type_count = len(labels.descriptors) ?
maybe introduce a function which adds a descriptor to a label which update type_count, or type_count can turn into a function which returns len(labels.descriptors) ?